draft-jasdips-regext-rdap-rpki.md

```diff
@@ -110,7 +116,7 @@ The Route Origin Authorization (ROA) object class can contain the following memb
   representing the not-valid-after date of the end-entity certificate for the ROA ([@!RFC6487, section 4])
 * autoRenewed -- a boolean indicating if the registered ROA is auto-renewed or not
 * events -- see [@!RFC9083, section 4.5]
-* links -- links ([@!RFC9083, section 4.2]) for "self", and "related" to IP network and IRR objects
+* links -- links ([@!RFC9083, section 4.2]) for "self", and "related" to IP network and IRR (when defined) objects
```

Comment:
maybe it is time to define the IRR objects. I would think only route and route6 are needed.

Reply:
> maybe it is time to define the IRR objects. I would think only route and route6 are needed.

Impressed with you catching this. :) Right, we could explore that after this draft has been introduced in regext. 

Although, folks may ask why have it in RDAP when the IRRd Whois service exists. I think I know the answer (same rationale as for RDAP replacing Whois) but would involve some convincing.